packages/Progressbar/Linear/LinearProgressbar.vue

```diff
@@ -1,36 +1,25 @@
 <template>
-  <progress :value="getValue" max="100" :class="{indeterminate, success, danger, warning, info}"></progress>
+  <div class="progressbar" :style="{ backgroundColor: `${backgroundColor}`}">
+    <div :style="{width: `${validateValue}%`, backgroundColor: `${color}`}" :class="{info,warning,danger,success,animate}" />
+  </div>
 </template>
 
 <script>
 export default {
   name: 'LinearProgressbar',
   props: {
-    value: {
-      type: Number,
-      default: 0
-    },
-    info: {
-      type: Boolean,
-      default: false
-    },
-    warning: {
-      type: Boolean,
-      default: false
-    },
-    danger: {
-      type: Boolean,
-      default: false
-    },
-    success: {
-      type: Boolean,
-      default: false
-    },
-    indeterminate: { type: Boolean, default: false }
+    value: { type: Number, default: 0 },
+    info: { type: Boolean, default: false },
+    warning: { type: Boolean, default: false },
+    danger: { type: Boolean, default: false },
+    success: { type: Boolean, default: false },
+    animate: { type: Boolean, default: false },
```

Comment:
I don't understand the `animate` property.
The animation should come from the value that is updated or it should come because we don't have any value and in that case, it's the `indeterminate` with an animation that is not 0 to 100% eg: https://codepen.io/AlexMaley/pen/ravVwQ